src/constants.js

```diff
@@ -34,6 +34,9 @@ exports.K = 20
 // Alpha is the concurrency for asynchronous requests
 exports.ALPHA = 3
 
+// How long individual query messages are allowed to take
```

Comment:
I believe RPCs already have a maximum timeout of 60s

Reply:
Yes, you're right, although it only applies when we expect to read a response. The 5minute was also a mistake, I am playing with lower timeouts and will post some metrics of provide calls I have been doing.